pkg/providers/tinkerbell/hardware/validator.go

```diff
@@ -43,6 +44,11 @@ func (mv *DefaultMachineValidator) Register(v ...MachineAssertion) {
 	mv.assertions = append(mv.assertions, v...)
 }
 
+var (
+	linuxPathRegex      = `^/|(/[\w-]+)+$`
```

Comment:
wonder if we should force either have the path start with '/dev' or if it doesn't prepend ourselves

Reply:
Made the check stricter with a `/dev/` prefix requirement.